docs/using-with-mapbox-gl.md

```diff
@@ -7,11 +7,29 @@ the [react-map-gl](https://uber.github.io/react-map-gl/#/) component,
 which is a React wrapper around mapbox-gl, and is the recommended companion
 to use deck.gl for most applications.
 
-
 ## About react-map-gl
 
 As shown in the examples, the `DeckGL` React component works especially
 well as the child of a React component such as react-map-gl that displays
 a map using parameters similar to the deck.gl Viewport (i.e. latitude,
 longitude, zoom etc). In this configuration your deck.gl layers will
 render as a perfectly synchronized geospatial overlay over the underlying map.
+
+## About Mapbox Tokens
+
+To show maps from a service such as Mapbox you will need to register with
+Mapbox and get a "token" that you need to provide to your map component.
```

Comment:
register with Mapbox to get a "token" needed to display any map content from them.